rplugin/python3/molten/moltenbuffer.py

```diff
@@ -223,8 +223,8 @@ def tick(self) -> None:
 
                 output.end_time = datetime.now()
 
-        if did_stuff:
-            self.update_interface()
+        # if did_stuff:
```

Comment:
Hmmm. Seeing this implementation, I think this should be configurable, but on by default. Just in case it causes performance problems for people